src/VisualStudio/CSharp/Impl/Debugging/CSharpBreakpointResolutionService.cs

```diff
@@ -15,29 +18,33 @@ namespace Microsoft.VisualStudio.LanguageServices.CSharp.Debugging
     [ExportLanguageService(typeof(IBreakpointResolutionService), LanguageNames.CSharp), Shared]
     internal partial class CSharpBreakpointResolutionService : IBreakpointResolutionService
     {
-        internal static async Task<BreakpointResolutionResult> GetBreakpointAsync(Document document, int position, CancellationToken cancellationToken)
+        /// <summary>
+        /// Returns null if a breakpoint can't be placed at the specified position.
```

Comment:
I think the "Async" in the name implies this refers to the return value of the task.